testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksCompatibilityCleanUp.java

```diff
@@ -37,52 +36,46 @@
 import static java.lang.String.format;
 import static java.lang.System.currentTimeMillis;
 import static java.util.Objects.requireNonNull;
-import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.stream.Collectors.toUnmodifiableList;
 
 public class TestDatabricksCompatibilityCleanUp
         extends ProductTest
 {
     private static final Logger log = Logger.get(TestDatabricksCompatibilityCleanUp.class);
-    private static final long SCHEMA_CLEANUP_THRESHOLD_SECONDS = (currentTimeMillis() / 1000) - DAYS.toSeconds(7);
+    private static final Instant SCHEMA_CLEANUP_THRESHOLD_SECONDS = Instant.now().minus(7, ChronoUnit.DAYS);
     private static final long MAX_JOB_TIME_MILLIS = MINUTES.toMillis(5);
 
     @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
     public void testCleanUpOldTablesUsingDelta()
-            throws Exception
     {
-        String hiveMetastoreUri = requireNonNull(System.getenv("HIVE_METASTORE_URI"), "Environment HIVE_METASTORE_URI was not set");
-        URI metastoreUri = URI.create(hiveMetastoreUri);
+        AWSGlueAsyncClientBuilder asyncGlueClientBuilder = AWSGlueAsyncClientBuilder.standard();
+        String awsRegion = requireNonNull(System.getenv("AWS_REGION"), "Environment AWS_REGION was not set");
+        asyncGlueClientBuilder.setRegion(awsRegion);
+        AWSGlueAsync glueClient =asyncGlueClientBuilder.build();
+
         long startTime = currentTimeMillis();
         List<String> schemas = onTrino().executeQuery("SELECT DISTINCT(table_schema) FROM information_schema.tables")
                 .rows().stream()
                 .map(row -> (String) row.get(0))
                 .filter(schema -> schema.toLowerCase(Locale.ROOT).startsWith("test") || schema.equals("default"))
                 .collect(toUnmodifiableList());
-        try (ThriftHiveMetastoreClient thriftHiveMetastoreClient = new ThriftHiveMetastoreClient(
-                Transport.create(HostAndPort.fromParts(metastoreUri.getHost(), metastoreUri.getPort()),
-                        Optional.empty(),
-                        Optional.empty(),
-                        100000,
-                        new NoHiveMetastoreAuthentication(),
-                        Optional.empty()),
-                metastoreUri.getHost())) {
-            // this is needed to make deletion of some views possible
-            onTrino().executeQuery("SET SESSION hive.hive_views_legacy_translation = true");
-            schemas.forEach(schema -> cleanSchema(schema, startTime, thriftHiveMetastoreClient));
```

Comment:
before the change, what did this code execute against?

Reply:
This code was never executed before in the context of Trino.
This code has been open sourced along with the other product tests, but not used until now.